Cargo.toml

```diff
@@ -25,7 +25,7 @@ bytebuffer = "2.3.0"
 criterion = "0.7"
 downcast-rs = "2.0.2"
 fast-str = "1.0.0"
-indexmap = "2.12.0"
+indexmap = { version = "2.12.0", features = ["serde"] }
```

Comment:
Should we turn on serde feature by default? Not important yet, but we will probably want to put all serde integration behind a feature flag.

Reply:
Agreed, behind flag